src/uct/ib/base/ib_device.h

```diff
@@ -123,6 +124,10 @@ typedef struct uct_ib_device {
     UCS_STATS_NODE_DECLARE(stats);
     struct ibv_exp_port_attr    port_attr[UCT_IB_DEV_MAX_PORTS]; /* Cached port attributes */
     unsigned                    flags;
+    uint8_t                     atomic_arg_size;
```

Comment:
these 4 vars should be either `atomic_arg_sizes`, `atomic_size_bitmap` or similar - to indicate it's a bitmap